props/saber_fett263_buttons.h

```diff
@@ -5645,32 +5645,36 @@ SaberFett263Buttons() : PropBase() {}
 
 #ifdef FETT263_SPECIAL_ABILITIES
       case EVENTID(BUTTON_NONE, EVENT_TWIST_RIGHT, MODE_ON | BUTTON_AUX):
-        if (!menu_) {
+        if (!menu_ && !swinging_) {
```

Comment:
Do you really want to update current_menu_angle_ when swinging?

Reply:
Going to look at just ignoring the DetectMenuTurn() while the saber is swinging altogether.  Just need to test, will update once I make sure it works as intended.